docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+
+## Variable
+
+### showInForm
+
+Check if variable is shown in the input form in draft mode.
+
+```scala
+showInForm(
+  variable: VariableDefinition,
+  executionResult: TemplateExecutionResult
+): Boolean
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `variable` | [`VariableDefinition`](#variabledefinition) | **Required.** A [`VariableDefinition` object](#variabledefinition) representing the variable. |
+| `executionResult` | [`TemplateExecutionResult`](#templateexecutionresult) | **Required.** The nested object returned from the [`execute` method](#execute) and [`executeForReview` method](#executeforreview). |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const allVariables = Openlaw.getVariables(executionResult.executionResult, {});
+const variable = allVariables[0]; // variable being checked is first variable in template
+Openlaw.showInForm(variable, executionResult.executionResult);
+```
+
+**Response**
+
+Returns `true` if variable is of type that should be displayed in the input form in draft mode (e.g., [input variables](/markup-language/#variables) like Text, Number, Date, Address, etc.).
+
+### getType
+
+Get variable type.
+
+```scala
+getType(
+  variable: VariableDefinition
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `variable` | [`VariableDefinition`](#variabledefinition) | **Required.** A [`VariableDefinition` object](#variabledefinition) representing the variable. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const allVariables = Openlaw.getVariables(executionResult.executionResult, {});
+allVariables.forEach(variable => {
+  Openlaw.getType(variable));
+});
+```
+
+**Response**
+
+Returns variable type as a string.
+
+Example
+
+```js
+"Text"
+"Date"
+"YesNo"
+"Number"
+```
+
+### getName
+
+Get name of a variable.
+
+```scala
+getName(
+  variable: VariableDefinition
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `variable` | [`VariableDefinition`](#variabledefinition) | **Required.** A [`VariableDefinition` object](#variabledefinition) representing the variable. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const allVariables = Openlaw.getVariables(executionResult.executionResult, {});
+allVariables.forEach(variable => {
+  Openlaw.getName(variable));
+});
+```
+
+**Response**
+
+Returns name of variable as a string.
+
+Example
+
+```js
+"Company Name"
+"Effective Date"
+"No Services"
+"Number of Shares"
```

Comment:
I think it is clearer to return just one string, otherwise it looks like an array without brackets (unless I am missing something)

Reply:
Removed the code block here and just referred to one example of `"Company Name"`.